src/PixiEditor/Models/DocumentModels/ActionAccumulator.cs

```diff
@@ -39,6 +39,7 @@ public void AddFinishedActions(params IAction[] actions)
         queuedActions.AddRange(actions);
         queuedActions.Add(new ChangeBoundary_Action());
         TryExecuteAccumulatedActions();
+        document.AutosaveViewModel.HintFinishedAction();
```

Comment:
You can still pass `ChangeBoundary_Action` into AddActions manually, and it won't get picked up. Although it is a bit hard to account for

Reply:
I decided that this doesn't really matter, saving during an updateable change shouldn't cause any issues apart from a potential lag spike, and this is fine for preventing the lag spike